src/dirtyfields/compat.py

```diff
@@ -14,14 +18,15 @@ def get_m2m_with_model(given_model):
 
 
 def is_buffer(value):
-    if sys.version_info < (3, 0, 0):
-        return isinstance(value, buffer)  # noqa
+    if PY3:
+        buffer_type = memoryview
```

Comment:
Probably it would be better to have ``buffer_type`` on module level because otherwise this condition would be executed on every call of ``is_buffer``.